sabnzbd/decoder.py

```diff
@@ -173,7 +173,7 @@ def run(self):
                         logging.info(logme)
```

Comment:
Please summarize in the commit comment the rationale for this. Explain what happened.
The actual code changes seem to be more than is covered by the commit title.